subsys/fb/Kconfig

```diff
@@ -13,12 +13,30 @@ menuconfig CHARACTER_FRAMEBUFFER
 
 if CHARACTER_FRAMEBUFFER
 
+
+choice
+	prompt "Fonts in use"
+	default CHARACTER_FRAMEBUFFER_USE_8X8_FONTS if DISPLAY_MCUX_ELCDIF
+	help
+	  Font that used in framebuffer
+
 config CHARACTER_FRAMEBUFFER_USE_DEFAULT_FONTS
-	bool "Use default fonts"
-	default y
+	bool "Use tiled fonts"
 	help
 	  Use default fonts.
 
+config CHARACTER_FRAMEBUFFER_USE_8X8_FONTS
+	bool "Use plain 8x8 fonts"
+	help
+	  Use 8x8 fonts.
+
+config CHARACTER_FRAMEBUFFER_USE_CST_FONTS
```

Comment:
the initial idea is just to leave an entry to those who want to customer, so that they can use this config to add source code.  such as
`zephyr_sources_ifdef(CONFIG_CHARACTER_FRAMEBUFFER_USE_CST_FONTS cfb_cstfonts.c`

Reply:
Do you really expect customers to add code directly into the main tree, rather than in their application code?  If so, then they can also modify `CMakeLists.txt`.  IMO this option really shouldn't be here.

Reply:
accept, removed.